config/crd/bases/exp.infrastructure.cluster.x-k8s.io_azuremachinepools.yaml

```diff
@@ -298,6 +310,22 @@ spec:
                 - sshPublicKey
                 - vmSize
                 type: object
+              userAssignedIdentities:
+                description: UserAssignedIdentities is a list of standalone Azure
+                  identities provided by the user The lifecycle of a user-assigned
+                  identity is managed separately from the lifecycle of the AzureMachinePool.
```

Comment:
I think I know what this means; the user is responsible for making the user assigned identity in Azure on their own. If this is the case, it would be great to link to some documentation about creating a user assigned identity. In other words, can be more specific and action oriented to help a user to figure this out?